Code/GraphMol/Canon.cpp

```diff
@@ -1086,69 +1060,69 @@ void canonicalizeFragment(ROMol &mol, int atomIdx,
                   "Corrupted stack. First element should be an atom.");
 
   // collect some information about traversal order on chiral atoms
-  bool *numSwapsChiralAtoms = (bool *)malloc(nAtoms * sizeof(bool));
-  CHECK_INVARIANT(numSwapsChiralAtoms, "failed to allocate memory");
-  memset(numSwapsChiralAtoms, 0, nAtoms * sizeof(bool));
```

Comment:
This one had been making me uncomfortable for some time...

Reply:
yeah, I wasn't sad to remove that